### PR TITLE
Doc improvements

### PR DESCRIPTION
**Pull request by panyx0718**
--parallel no longer exist
add a simple runnable example.

### CODE REVIEW

File: benchmark/fluid/README.md [hunk 1]
```diff
@@ -31,15 +31,15 @@ Currently supported `--model` argument include:
...
         ```
     * start trainers:
         ```bash
-        PADDLE_TRAINING_ROLE=PSERVER PADDLE_PSERVER_PORT=7164 PADDLE_PSERVER_IPS=127.0.0.1 PADDLE_TRAINERS=1 PADDLE_CURRENT_IP=127.0.0.1 PADDLE_TRAINER_ID=0 python fluid_benchmark.py --model mnist --parallel 0 --device GPU --update_method pserver
+        PADDLE_TRAINING_ROLE=TRAINER PADDLE_PSERVER_PORT=7164 PADDLE_PSERVER_IPS=127.0.0.1 PADDLE_TRAINERS=1 PADDLE_CURRENT_IP=127.0.0.1 PADDLE_TRAINER_ID=0 python fluid_benchmark.py --model mnist --device GPU --update_method pserver
         ```
 * Run distributed training using NCCL2
     ```bash
-    PADDLE_PSERVER_PORT=7164 PADDLE_TRAINER_IPS=192.168.0.2,192.168.0.3  PADDLE_CURRENT_IP=127.0.0.1 PADDLE_TRAINER_ID=0 python fluid_benchmark.py --model mnist --parallel 0 --device GPU --update_method nccl2
+    PADDLE_PSERVER_PORT=7164 PADDLE_TRAINER_IPS=192.168.0.2,192.168.0.3  PADDLE_CURRENT_IP=127.0.0.1 PADDLE_TRAINER_ID=0 python fluid_benchmark.py --model mnist --device GPU --update_method nccl2
```
Comment (typhoonzero) [inline benchmark/fluid/README.md]: Thought this already have some fixes, can you please update this PR?

Comment (panyx0718) [reply benchmark/fluid/README.md]: Done

File: benchmark/fluid/run_dist_benchmark_example.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+
+PADDLE_TRAINING_ROLE=PSERVER PADDLE_PSERVER_PORT=7164 PADDLE_PSERVER_IPS=127.0.0.1 PADDLE_TRAINERS=2 PADDLE_CURRENT_IP=127.0.0.1 PADDLE_TRAINER_ID=0 python fluid_benchmark.py --model resnet --device GPU --update_method pserver --iterations=10000 &
```
Comment (typhoonzero) [inline benchmark/fluid/run_dist_benchmark_example.sh]: Seems the style check on travis ci didn't pass. And, I think maybe it's a good idea to put this in the `README.md`, for a distributed training example?

**Comment by typhoonzero** [comment]
Thanks for updating this PR!

File: benchmark/fluid/run_dist_benchmark_example.sh (hunk 2 above)
Comment (panyx0718) [reply benchmark/fluid/run_dist_benchmark_example.sh]: sure. perhaps tell user to start ps first and maybe sleep for a while?

**Pull request closed without merging**